tensorflow/lite/micro/tools/make/ext_libs/cmsis_nn.inc

```diff
@@ -8,7 +8,17 @@ ifneq ($(filter cmsis-nn,$(ALL_TAGS)),)
     THIRD_PARTY_DOWNLOADS += \
       $(eval $(call add_third_party_download,$(CMSIS_URL),$(CMSIS_MD5),cmsis,patch_cmsis))
 
-    CMSIS_PATH = $(MAKEFILE_DIR)/downloads/cmsis/
+    ifeq ($(CMSIS_PATH),)
+      CMSIS_PATH = $(MAKEFILE_DIR)/downloads/cmsis/
+    else
+      # Currently needed when jacking in an unpatched CMSIS-NN.
+      # TODO: Remove when CMSIS-NN no longer need to be patched.
```

Comment:
I gather that this is needed to allow @ml-0 to use CMSIS-NN that is not part of the downloads? Let's add an issue to the TODO to give more context on why this is needed.

Something like:
```
# TODO(#1234): Some use-cases need an upatched CMSIS-NN and these include paths
#  allow for that. In reality, patching is only needed for project generation (e.g. Arduino) so
#  it may be possible to always use an unpatched CMSIS-NN if there is a different approach
#  for project generation.
```

Reply:
I was not trying to use an unpatched CMSIS but I accidentally used an incompletely patched CMSIS because of #44001. After #44001 is merged, I can use cmsis_nn.inc as @mansnils provided it. Anyway, the option of using an unpatched CMSIS might be useful.

Reply:
ok, I'll let @mansnils decide what he prefers here -- I'm ok with allowing for unpatched CMSIS if the steps are documented in the README. Or removing that option for now and adding support on an as needed basis.

Reply:
OK, adding some info to the README.